schema/openeox_shell_json_schema.json

```diff
@@ -0,0 +1,70 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://docs.oasis-open.org/openeox/tbd/openeox_shell_json_schema.json",
+    "title": "EOL and EOS Information",
+    "description": "A draft schema for representing End-of-Life (EOL) and End-of-Support (EOS) information in OpenEoX.",
+    "type": "object",
+    "$defs": {
+        "productName_t": {
+            "title": "Product name",
+            "description": "Contains the name of the product.",
+            "type": "string",
+            "minLength": 1
+          },
+          "productVersion_t": {
+            "title": "Product version",
+            "description": "Contains the version or release of the product.",
+            "type": "string",
+            "minLength": 1
+          },
+        "supplierName_t": {
+            "title": "Supplier name",
+            "description": "Contains the name of the supplier or service provider.",
+            "type": "string",
+            "minLength": 1
+        },
+        "schema_t": {
+            "title": "OpenEoX Shell schema",
+            "description": "Specifies the schema the JSON object must be valid against.",
+            "type": "string",
+            "enum": [
+                "https://docs.oasis-open.org/openeox/tbd/openeox_shell_json_schema.json"
```

Comment:
With path separators:
`https://docs.oasis-open.org/openeox/tbd/schema/shell.json`

Reply:
addressed in [b57311f](https://github.com/oasis-tcs/openeox/pull/10/commits/b57311f3895a5208794ebec25f72fbf200fb4d3e)